tests/system/small/test_dataframe.py

```diff
@@ -2074,6 +2075,37 @@ def test_series_binop_axis_index(
     assert_pandas_df_equal(bf_result, pd_result)
 
 
+@skip_legacy_pandas
+@pytest.mark.parametrize(
+    ("input"),
+    [
+        ((1000, 2000, 3000)),
+        (pd.Index([1000, 2000, 3000])),
+        (bf_indexes.Index([1000, 2000, 3000])),
+        (pd.Series((1000, 2000), index=["int64_too", "float64_col"])),
+        (series.Series((1000, 2000), index=["int64_too", "float64_col"])),
+    ],
+    ids=[
+        "tuple",
+        "pd_index",
+        "bf_index",
+        "pd_series",
+        "bf_series",
+    ],
+)
+def test_listlike_binop_axis_1(scalars_dfs, input):
```

Comment:
Should we add axis=1 to some existing tests for various binary ops?

Reply:
axis=1 completely changes alignment so the objects would just fail to align on this axis for existing tests. I'm not too worried about the different ops breaking, the actual scalar op compilation doesn't change for axis=1, the big worry is alignment failing

Reply:
Oh, right, the existing objects wouldn't wouldn't work. Okay, sounds good.